src/components/statefilter.js

```diff
@@ -8,9 +8,9 @@ export default function StateFilter(props) {
     return (
         <Pane display="flex" alignItems="center" padding={8} marginX={majorScale(3)}>
             <SelectMenu
-                title="Select State"
+                title="Select City"
                 onFilterChange={(filter) => setFilter(filter)}
-                options={['New York', 'New Jersey'].map((label) => ({ label, value: label }))}
+                options={['Charlotte', 'Los Angeles','Houston', 'Indianapolis', 'Jacksonville','Chicago', 'New York City', 'Philadelphia','Phoenix', 'Seattle'].map((label) => ({ label, value: label }))}
```

Comment:
Abstract the options array to a variable outside the scope of this component. Can keep the map() on this line exactly the same

Reply:
changed